lib/drivex.js

```diff
@@ -75,6 +75,26 @@ DriveX.prototype = {
       }
     });
   },
+    /**
+   * wraps Selenium WebElement.elementIsNotVisible
+   * @param locator {LocatorJSON}
+   * @param el {WebElement}
+   * @returns {Promise} resolves to true or rejected
+   */
+  notVisible: function (locator, el) {
```

Comment:
Since this essentially duplicates the functionality of the `visible` method, can't we just use the `visible` method anywhere we'd use this `notVisible` method?